_storage/write-to-read.md

```diff
@@ -63,3 +63,16 @@ demonstrate that they control _that_ address. This is achieved via the
 authentication token which is a message _signed_ by the private key associated
 with that address. The message itself is a challenge text, returned via the
 `/hub_info/` endpoint.
+
+Reads can be done by everybody. The URLs to a user's app data are in a canonical location in their profile. For example, here's how you would get data from the
+￼ [Banter](https://banter.pub/) app, stored under the Blockstack ID `gavin.id`.
+￼
+￼ 1. Get the bucket URL
+￼ `bash ￼ $ BUCKET_URL="$(curl -sL https://core.blockstack.org/v1/users/gavin.id | jq -r '."gavin.id"["profile"]["apps"]["https://banter.pub"]')" ￼ $ echo "$BUCKET_URL" ￼ https://gaia.blockstack.org/hub/16E485MVpR3QpmjVkRgej7ya2Vnzu3jyTR/ ￼`
```

Comment:
I think this is a useful example and provides good context. Only issue is that `jq` is probably not installed on most machines. 
@friedger thoughts?